Sources/SPMTestSupport/MockPackage.swift

```diff
@@ -73,6 +73,7 @@ public struct MockPackage {
         name: String,
         platforms: [PlatformDescription] = [],
         identity: String,
+        alternativeURLs: [String]? = .none,
```

Comment:
Is there a semantic distinction between `[]` and `.none` here?  i.e. should it be maybe be non-optional?

Reply:
its a style thing, I guess (not sure about memory implication, empty array maybe also memory efficient). I personally prefer to use optionals when the data is semantically optional